doc/guides/maintaining-V8.md

```diff
@@ -1,10 +1,10 @@
-# Maintaining V8 in Node
+# Maintaining V8 in Node.js
 
 # Background
 
-V8 follows the Chromium release schedule. The support horizon for Chromium is very different from the support horizon that Node.js needs to provide to its users. As a result Node.js needs to support a version of V8 for quite a bit longer than what upstream needs to support. Since V8 doesn't have an LTS supported branch, there is no official process around how the V8 branches in Node are maintained.
+V8 follows the Chromium release schedule. The support horizon for Chromium is very different from the support horizon that Node.js needs to provide to its users. As a result Node.js needs to support a version of V8 for quite a bit longer than what upstream needs to support. Since V8 doesn't have an LTS supported branch, there is no official process around how the V8 branches in Node.js are maintained.
```

Comment:
For simple text paragraphs, I've now wrapped to 80 chars.

(I didn't do it for lists and anything with interesting formatting because I don't trust my editor to not break something. If anyone feels strongly enough about it and wants to wrap the bulleted lists and paragraphs with links and whatnot, by all means, feel free to make the change and push to this branch.)